### PR TITLE
ssh through specific interface

### PR DESCRIPTION
**Pull request by prudmirya**
this patch enable ssh through specific interface, when u have more than
one interface u can specify specific interface to ssh, and it remove
pep8 errors

Signed-off-by: prudhvi <mprudhvi@linux.vnet.ibm.com>

### CODE REVIEW

File: virttest/remote.py [hunk 1]
```diff
@@ -262,6 +268,7 @@ def remote_login(client, host, port, username, password, prompt, linesep="\n",
 
     if verbose:
         logging.debug("Login command: '%s'", cmd)
+    print cmd
```
Comment (xutian) [inline virttest/remote.py]: Excess line

File: virttest/remote.py [hunk 2]
```diff
@@ -213,7 +213,7 @@ def handle_prompts(session, username, password, prompt, timeout=10,
 
 def remote_login(client, host, port, username, password, prompt, linesep="\n",
                  log_filename=None, timeout=10, interface=None,
-                 status_test_command="echo $?", verbose=False):
+                 status_test_command="echo $?", verbose=False, bind_ip=False):
```
Comment (balamuruhans) [inline virttest/remote.py]: it is better to have `bind_ip=None`, it is not boolean and expects interface ip

File: virttest/remote.py [hunk 3]
```diff
@@ -891,8 +898,8 @@ def copy_files_from(address, client, username, password, port, remote_path,
     :param verbose: If True, log some stats using ``logging.debug`` (RSS only)
     :param timeout: The time duration (in seconds) to wait for the transfer to
                     complete.
-    :param interface: The interface the neighbours attach to (only use when using ipv6
-                linklocal address.)
+    :param interface: The interface the neighbours attach to (ony
```
Comment (balamuruhans) [inline virttest/remote.py]: %s/ony/only

**Comment by xutian** [comment]
ACK

**Comment by xutian** [comment]
Merged, thanks @prudmirya's contribution.

**Pull request merged**